src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+  const TOTAL_PRICE = days * PRICE_PER_DAY;
```

Comment:
It is customary to name static variables with a capital letter, if something is calculated in the variable, it is better to name it using the usual camelCase.